### DISCUSSION TITLE
Advice wanted: Clustering of relevant items and when to stop

### DISCUSSION THREAD

**Discussion by JantineW** [asreview/asreview#1473] [Q&A] [answered]
Hello everyone!
We are looking for some advice. As you probably know, there are many options for deciding when to stop screening. We decided that we would stop after 33% (based on Van de Schoot et al., 2021). However, during the reviewing process, we saw clustering of relevant items. And now that we are at 33%, there still seem to be relevant items - maybe another cluster is still coming. The analytics look like this at the moment:
![image](https://github.com/asreview/asreview/assets/137302417/85727936-d5de-483b-ae97-7e3e584128df)

What do you think would be wise to do at this point? Stop screening anyways? Or continue? But then when would we stop?
Any advice is very much appreciated. Thanks in advance!

**Comment by Rensvandeschoot** [comment]
See the reply in this thread: https://github.com/asreview/asreview/discussions/1470#discussioncomment-6184842

**Reply by Rensvandeschoot** [reply, marked as answer]
see also the pre-print: https://psyarxiv.com/c93gq

**Reply by JantineW** [reply]
Great, thank you!
